products/uosv20/product.yml

```diff
@@ -0,0 +1,27 @@
+product: uosv20
+full_name: Uniontech OS V20
+type: platform
+
+benchmark_id: UOS-V20 
+benchmark_root: "../../linux_os/guide"
+
+profiles_root: "./profiles"
+
+pkg_manager: "yum"
+
+init_system: "systemd"
+
+cpes_root: "../../shared/applicability"
+cpes:
+  - uos:
+    name: "cpe:/o:uos:uniontech_os:v20"
+      title: "Uniontech OS V20"
+      check_id: installed_OS_is_uosv20
+
+# Mapping of CPE platform to package
+platform_package_overrides:
+  login_defs: "shadow-utils"
+
+reference_uris:
+  cis: 'https://www.cisecurity.org/benchmark/centos_linux'
```

Comment:
Please remove this line as this product doesn't have a CIS benchmark profile yet.